### ISSUE TITLE
Fichier evaluer.sh

### ISSUE THREAD

**Issue by albache** [guyfrancoeur/INF3135_H2019_TP2#22]
Bonjour a tous,

Pour le fichier evaluer.sh, les commandes mentionnées, seront-elles executées en ordre ou non?

**Comment by TristanEli** [comment]
Oui, comme pour le tp1 elles seront exécutées une après l'autre.

**Comment by btaousoussema** [comment]
Salut, 

Non, il a dit en classe que ce n'était pas nécessairement en ordre et qu'il fallait se préparer en conséquences (être sûr que c'est le plus automatisé possible).

**Comment by Matheus-Brum** [comment]
Pour préciser ce que @btaousoussema a dit, les commandes peuvent être éxécutées de la fin jusqu'au début plutôt que de commencer du début jusqu'à la fin.
